editor/debugger/editor_debugger_inspector.cpp

```diff
@@ -55,13 +55,8 @@ bool EditorDebuggerRemoteObject::_get(const StringName &p_name, Variant &r_ret)
 }
 
 void EditorDebuggerRemoteObject::_get_property_list(List<PropertyInfo> *p_list) const {
-	p_list->clear(); // Sorry, no want category.
+	p_list->clear(); // Clear list to get rid of categories.
 	for (const PropertyInfo &prop : prop_list) {
-		if (prop.name == "script") {
-			// Skip the script property, it's always added by the non-virtual method.
-			continue;
-		}
-
 		p_list->push_back(prop);
 	}
 }
```

Comment:
Doesn't seem amount to copying a `List<PropertyInfo>` to another `List<PropertyInfo>` without modification? Maybe this can be bypassed, unless that's needed to get a separate mutable copy?

Reply:
I think it might have been intended to get a separate mutable copy, maybe @Faless remembers since he implemented #36244.